Src/DataTypes/Gpkg.cs

```diff
@@ -150,6 +150,7 @@ public override void Cleanup()
 
         public override bool Exists()
         {
+            Console.WriteLine($"Checking if exists, gpkg: {this.path}");
```

Comment:
should use logger if this will be add to service ( log4net maybe ?)

Reply:
When transforming to a worker, this will be part of the proccess.